benchmarks/record_batch_compose.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+from __future__ import print_function
+import perf
+from kafka.record.memory_records import MemoryRecordsBuilder
+import itertools
+import random
+import hashlib
+import os
+
+
+DEFAULT_BATCH_SIZE = 1600 * 1024
+KEY_SIZE = 6
+VALUE_SIZE = 60
+TIMESTAMP_RANGE = [1505824130000, 1505824140000]
+
+# With values above v1 record is 100 bytes, so 10_000 bytes for 100 messages
```

Comment:
`10,000 bytes`?

Reply:
Yes, it's a habit from developing under 3.6. It allows to write`x = 10_000_000` and will just ignore underscores.